snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-0

```diff
@@ -35,6 +35,32 @@
         ]
       }
     },
+    "metrics": {
+      "description": "Key-value pairs of metrics",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metric attribute",
+        "properties": {
+          "key": {
+            "description": "The key for this attribute",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this attribute",
+            "type": ["string", "boolean", "integer", "null"],
```

Comment:
**question:** What would be the cases when the value is `null`? Shouldn't we always get a value for a metric?

Reply:
For the `"metrics"` section, I can be sure we don't produce nulls, so I'll remove it.

Reply:
Fixed